src/Controller/Admin/BlogController.php

```diff
@@ -30,7 +30,7 @@
  * purposes. However, in your real Symfony application you should use any of the
  * existing bundles that let you generate ready-to-use backends without effort.
  *
- * See http://knpbundles.com/keyword/admin
+ * See https://symfony.com/bundles/EasyAdminBundle/current/index.html
```

Comment:
I've replaced this link by `https://symfony.com/bundle` There are several admin bundles, so it's not fair to link specifically to one of them 👍